themes/bootstrap3/templates/search/list-list.phtml

```diff
@@ -1,7 +1,20 @@
+<?php
+  $listClasses = ['record-list'];
+  if ($this->results->getOptions()->loadResultsWithJsEnabled()) {
+    $this->headScript()->appendFile('search.js');
+    // Ensure OpenURL support is available even if none of the results on the initial
+    // page has one:
+    $this->headScript()->appendFile('openurl.js');
+    $listClasses[] = 'js-record-list';
+  }
+  if ($class = $this->resultsClass ?? '') {
+    $listClasses[] = $this->escapeHtmlAttr($class);
```

Comment:
Would it be more clear to escape the imploded list below, or else to add a variable suffix to clarify that these are already escaped?

Reply:
I refactored this out. :)